Networking/Networking/Model/Order.swift

```diff
@@ -28,6 +28,7 @@ public struct Order: Decodable, GeneratedCopiable {
     public let shippingTax: String
     public let total: String
     public let totalTax: String
+    public let paymentMethodID: String
```

Comment:
As far as I can tell, it looks like `payment_method` is not optional. 👍  

It's just scary that it's saved in an optional meta database in WordPress. 😬